db/migrations/20230912062551_repo_state_2.up.sql

```diff
@@ -0,0 +1,2 @@
+-- Add up migration script here
+alter table repo_state add column is_branch integer;
```

Comment:
I just thought of this, but is it valid to have a `NULL` `is_branch`? Is that just a revision?

If it isn't valid, I'd consider using a `not null`(https://www.sqlite.org/syntax/column-constraint.html) and/or a `default` value [constraint](https://www.sqlite.org/syntax/column-constraint.html).

Reply:
It's possible for `is_branch` to be null. This would happen when a user has existing entries in their db and they run broker again which would trigger the new db migration. All the entries would have a null value for that column in that case.